project-base/app/src/FrontendApi/Resolver/Products/ProductsQuery.php

```diff
@@ -162,6 +162,7 @@ public function productsWithOverlyingEntityQuery(Argument $argument, ResolveInfo
         }
 
         if ($argument['flagSlug'] !== null) {
+            /** @var \App\Model\Product\Flag\Flag $flag */
```

Comment:
uncaught App namespace here

Reply:
(in the later commit, this code is moved to the fe API package but the inline annotation is preserved

Reply:
a9e191a9a0